daemon/api_sideload_n_try.go

```diff
@@ -74,52 +95,32 @@ func sideloadOrTrySnap(c *Command, body io.ReadCloser, boundary string, user *au
 	systemRestartImmediate := isTrue(form, "system-restart-immediate")
 
 	// find the file for the "snap" form field
-	var snapBody multipart.File
+	var tempPath string
 	var origPath string
 out:
-	for name, fheaders := range form.File {
+	for name, fheaders := range form.FileHeader {
```

Comment:
Then with SnapUploadForm we would just use the TempSnapFile field, rather than iterating over a map and then over a list picking up the first entry only.